src/sortedvector.jl

```diff
@@ -5,13 +5,13 @@ A wrapper for sorted vectors, designed for fast unions.
 
 # Constructor
 
-    SortedVector(data::AbstractVector; already_sorted=false)
+    SortedVector(data::AbstractVector; sorted=false)
```

Comment:
I liked `already_sorted` because it clearly refers to the input, whereas `sorted=false` feels very wrong for a `SortedVector`

Reply:
Yeah, I see the issue now. 👍